src/components/controllers/page/index.js

```diff
@@ -1,131 +1,19 @@
 import { Fragment, h } from 'preact';
-import { useEffect, useState, useMemo, useRef } from 'preact/hooks';
+import { useEffect, useMemo } from 'preact/hooks';
 import cx from '../../../lib/cx';
 import ContentRegion from '../../content-region';
-import { getContentOnServer, getContent } from '../../../lib/content';
 import config from '../../../config.json';
 import style from './style.module.css';
 import Footer from '../../footer';
 import Sidebar from './sidebar';
 import Hydrator from '../../../lib/hydrator';
 import EditThisPage from '../../edit-button';
-import {
-	getPrerenderData,
-	InjectPrerenderData
-} from '../../../lib/prerender-data';
+import { InjectPrerenderData } from '../../../lib/prerender-data';
 import { isDocPage } from '../../../lib/docs';
 import { useStore } from '../../store-adapter';
 import { AVAILABLE_DOCS } from '../../doc-version';
 import { Time } from '../../time';
-
-const getContentId = route => route.content || route.path;
-
-/**
- * Set `document.title`
- * @param {string} title
- */
-export function useTitle(title) {
-	useEffect(() => {
-		if (title) {
-			document.title = `${title} | ${config.title}`;
```

Comment:
I feel like this was meant to be an `||`? `Getting Started | Preact: Fast 3kb React alternative with the same ES6 API. Components & Virtual DOM.` is quite a bit of text for a title, but it could make sense as a fallback?